TurtleShroom Productions II

TurtleShroom opened the pull request:
THERE ARE NO BREAKS ON THIS TRAIN

TurtleShroom commented:
@loup99 

Ah'ight Loup, I've produced the next batch of alterations for your consideration. I await your commentary as soon as you can give it. I hope to be a very productive and efficient contributor to one of the greatest CK2 Mods to ever exist!

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -1062,7 +1062,6 @@ e_hunnic_empire = {
 	allow = {
 		culture = hunnic
 		prestige = 5000
-		religion_group = pagan_group
```
This I'd like to keep, as a check.

TurtleShroom replied:
@loup99 

No, no. ANY Hun should be able to reform the Hunnic Empire. The idea that someone loses their identity then they find Jesus is silly. Christian Mongols can still hold the Nerge and steppe nomads don't stop riding when they find Jesus, so why should the Huns?

I am willing, though, to exclude Merchant Republics.

loup99 replied:
OK.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -952,13 +952,19 @@ d_goths = {
+			culture = ostrogothic
+		}
+		controls = 560
+		prestige = 500
```
This should stay as "always = no"

TurtleShroom replied:
@loup99 

Understood. I thought "always = no" meant it could never be formed. I understand what it means, now. I'll add it back.

loup99 replied:
There must have been a misunderstanding. always = no does mean that it can't be formed, which is intentional design.

TurtleShroom replied:
@loup99 

The point here was that Gothic characters can restore Gothic duchies (etc.). It's a shame that those Titular Titles die out so easily, so I wanted to put in a reasonable fix for it. Likewise, Angle cultured people should be able to restore Angloland, and so on.

loup99 replied:
Those titles are supposed to die when the characters settle, and cause bugs otherwise.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -997,16 +1004,18 @@ d_ostrogoths = {
 	
 	culture = ostrogothic
-	# tribe = yes
+	tribe = yes
```
Not a desired change.

TurtleShroom replied:
So be it.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -997,16 +1004,18 @@ d_ostrogoths = {
-		always = no
+		culture = ostrogothic
+		prestige = 500
+		controls = 456
```
This should stay as "always = no"

TurtleShroom replied:
Understood

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -1020,11 +1029,13 @@ d_thracian_goths = {
-		always = no
+		culture = ostrogothic
+		culture = gothic
+		controls = 497
```
This should stay as "always = no"

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -1038,7 +1049,9 @@ d_prus = {
-		always = no
+		culture = prussian
+		prestige = 1000
+		martial = 12
```
This should stay as "always = no"

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -12055,6 +12055,8 @@ e_tartaria = {
 				b_mazongshan = {
 				}
+				b_bubba = {
+				}
```
Why did you add this?

loup99 commented on WTWSMS/events/WtWSMS_holy_orders_events.txt:
```diff
@@ -46,18 +46,18 @@ narrative_event = {
 				random_traits = yes
 				dynasty = random
-				religion = root_west_pagan_reformed
+				religion = root_west_pagan
```
Why would the character be non-reformed?

TurtleShroom replied:
That was an onversight.

loup99 commented on WTWSMS/events/WtWSMS_holy_orders_events.txt:
```diff
@@ -86,18 +86,19 @@ narrative_event = {
 				random_traits = yes
 				dynasty = random
-				religion = semitic_pagan_reformed
+				religion = semitic_pagan
```
Should remain semitic_pagan_reformed

TurtleShroom replied:
Why in the HECK should he follow a reformed religion that doesn't exist?!

TurtleShroom replied:
@loup99 

Why in the HECK should he follow a reformed religion that doesn't exist?!

loup99 replied:
Because it only appears when it does exist.

TurtleShroom replied:
@loup99 

No, I've seen it spawn without Semitic Paganism ever reforming. It spawns every time, I know this because I personally tested Root West Paganism after you implemented my upgrades to it.

Semitic Paganism announces it Reformation, just like every other Pagan religion.

loup99 replied:
How is that related to my point though?

TurtleShroom replied:
Under your code, Reformed Semitic Pagans spawn no matter what, even when they don't exist in-game. That doesn't make any sense.

loup99 replied:
Because that event only happens if they are reformed?

loup99 commented on WTWSMS/events/WtWSMS_holy_orders_events.txt:
```diff
@@ -86,18 +86,19 @@ narrative_event = {
-				religion = semitic_pagan_reformed
-				culture = bedouin_arabic
+				religion = root_west_pagan_reformed
+				culture = random
```
Why Root West instead of Semitic?

loup99 commented on WTWSMS/history/titles/k_mimamsa.txt:
```diff
@@ -0,0 +1,24 @@
+
+511.11.28 = {
+	holder = 0
+}
```
Why did you add the character 0 as a holder?

TurtleShroom replied:
@loup99 

I thought Character Zero means "generate a random character". What does it actually mean?

loup99 replied:
I have always thought it was the character with the ID 0.

loup99 commented on WTWSMS/localisation/WtWSMS_titles.csv:
```diff
@@ -1110,72 +1110,74 @@ e_egypt;Egyptian Empire;Empire 
+b_rima;Rima;;;;;;;;;;x
+b_hira;Hira;;;;;;;;;;x
+DRUHTINAZ;Druhtinaz;;;;;;;;;;x
+DRUHTINAZ_FOA;Great Warlord;;;;;;;;;;x
```
Not sure what you did here, but it will have to be overwritten.

TurtleShroom replied:
@loup99 

Did you see the notes? "Druht**ī**naz" is a High German/Protogerman/Gothic title meaning "suffix for a _leader of a war band_". It's a flavor title!

loup99 replied:
Oh I missed that, but did you modify anything else?

TurtleShroom replied:
Not in that file.

loup99 replied:
OK.

loup99 commented on ChangeLog.txt:
```diff
@@ -40,6 +40,13 @@ MINOR:
 - Luciferians now praise the Dawn Star and reject ignorance
 - Taoists can no longer hunt
+- Fixed issue with Root West Reformation where Anxi fails to get a working temple, preventing instant Reformation
+- Mimamsa's "incredibly powerful priests" are now actually powerful, but due to the hardcoding of Autocephalacy, which would be realistic for them, a temporary implementation of the Antipope and Investiture mechanics exists to represent the struggle between the Indian kingdoms and their Mimamsa clergy, who vie for control over the monastaries of the realm
```
Defining patriarches will be possible in the next update. Antipope and Investiture mechanics are not fitting for this religion.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -4811,6 +4814,12 @@ indian_group = {
+		investitutre = yes
+		can_grant_invasion_cb = yes
+		can_grant_claim = yes
+		can_retire_to_monastery = yes		
```
As mentioned above.

TurtleShroom replied:
Understood

TurtleShroom commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -984,7 +990,8 @@ d_venedi = {
 	allow = {
-		always = no
+		culture = venedi
+		prestige = 500
```
Understood

TurtleShroom commented:
@loup99 

Loup, I removed the Mimamsa edits you asked to remove and I fixed the "always = no" issue. Thank you for the advice.

This edit also upgrades one of the most interesting heresies in your Mod: followers of  Wahhabism (the sect of ISIL, Al-Qaeda, and Saudia Arbia) now have their own mechanics and are extremely aggressive and zealous! The religion that has constantly brought the West to its knees DESERVES an upgrade from being a simple heresy clone. I aspire to make a Decision where you can proclaim yourself the Wahhabi Caliph and get a strong claim on the Empires of Arabia and Persia. The Wahhabi Caliphate will either use the ISIL flag or the original black flag of the First Caliphate.

loup99 commented on WTWSMS/localisation/WtWSMS_religions.csv:
```diff
@@ -158,7 +167,9 @@ twelver_DESC;Twelver Shia or the Ithna'ashariyyah' is the largest branch of Shia
-wahhabi_DESC;Wahhabism is an ultra-conservative branch or sect of Sunni Islam. It is a religious movement among fundamentalist Islamic believers, with an aspiration to return to the earliest fundamental Islamic sources of the Quran and Hadith, with inspiration from the teachings of Medieval theologian Ibn Taymiyyah and early jurist Ahmad ibn Hanbal.;;;;;;;;;;;x
+wahhabi_DESC;Wahhabism is an ultra-conservative, extremist, violent branch or sect of Sunni Islam. It is a religious movement among fundamentalist Islamic believers, with an aspiration to return to the earliest fundamental Islamic sources of the Quran and Hadith, with inspiration from the teachings of Medieval theologian Ibn Taymiyyah and early jurist Ahmad ibn Hanbal. Wahhabis refuse to tolerate ...
+wahhabi_adj;Wahhabi;;;;;;;;;;;x
+wahhabi_FEATURES;�Y*�! Wahabbis are extremely aggressive and zealous \n�Y*�! Wahhabis have exceptional fighting abilities \n�Y*�! Wahhabis experience boosts in morale and army tactics \n�Y*�! Wahhabis have an attrition advantage in their homelands due to their scorched-earth tactics and use of unconventional warfare;;;;;;;;;;;x
```
Wahhabism in the mod doesn't fit with that description at start, so I think we will have to revert those changes.

loup99 commented on WTWSMS/common/religions/00_religions.txt:
```diff
@@ -1408,26 +1415,58 @@ muslim = {
 		uses_jizya_tax = yes
 	}
-	wahhabi = {
+	wahhabi = {  #this is the extremist sect practiced by ISIL, Al-Qaeda, and Saudi Arabia to this day, as such, it is extremely violent, as it remains totally unchanged from its inception in the seventh and eighth centuries
```
Wahhabism in the mod doesn't fit with that description at start, so I think we will have to revert those changes.
 Add a line note

TurtleShroom replied:
@loup99 

I believe that to be incorrect. Wahhabis have always been ultra-conservative and extreme. They're basically the Puritans of Islam. I'm very proud of this addition; I consider it innovative, and I tried to balance the benefits the religion gives instead of just making them crazed. It gives players the chance to be total villains, and it fits in with Islam's history. It's a heresy I'd want to play!

From what I know about Wahhabism, it was a heresy that started several decades after Muhammed's death, in either the 700s AD or the 800s AD. Wahhabis saw the unusual tolerance practiced by the Mutalazites and other more "enlightened" Muslims and rejected any peace with the infidels. They holed up deep in Arabia and Mesopotamia and constantly existed as a thorn in the side of every Muslim empire for centuries on end. They have warred everything that didn't conform to them for centuries.

Once a minority, the Wahhabi sect of Islam is one of the most powerful, most influential, and most violent sects of Islam to ever exist (they only became powerful a few decades ago).

This is a very accurate description of the Wahhabi religious sect. Even though it appears as a heresy immediately, that doesn't mean it shouldn't be treated as it was known for when it actually spawned, two hundred years later.

loup99 commented:
As mentioned, Wahhabism in the mod doesn't exactly fit with that. I think we will have to put the changes on hold until further ado.

TurtleShroom commented:
Wahhabism has not changed since it has existed. There's no way it doesn't fit with the Mod, unless I am confusing it with Salafism!

TurtleShroom commented:
@loup99 

Wahhabism has not changed since it has existed. There's no way it doesn't fit with the Mod, unless I am confusing it with Salafism!

TurtleShroom commented:
@loup99 

What exactly do you want to discuss? I have no problem talking about the changes.

(Sorry for the rapid fire posts. I don't know how to reply very well.)

loup99 commented:
No, just that you write what you plan on doing before doing it, so we can discuss it then. If I want you to not touch Wahhabism for now (you can on your personal version), it is because the Rise of Islam content is not in yet, and Wahhabism will need specific attention.

TurtleShroom commented:
@loup99 

Uh, okay. I don't see why we can't just leave it there right now, so we don't have to add it later. Is there anything there that will conflict with your plans? If there is any form of conflict, I'll gladly clean it out.

As for writing my plans, that's the problem: _I don't have any plans._ I'm adding as I go.

loup99 commented:
You can always keep the changes, but I would like to keep it out of the main branch for now.

TurtleShroom commented:
@loup99 

I'll take them out if I must, but I really don't understand what's wrong with keeping the changes that do not conflict with your vision. What exactly _is_ your plan for Wahhabism, anyway? I'd like to get an idea of it so I can understand why you object to my changes. It's strange, in my opinion, that these changes could somehow conflict with your plans.

Is it the description I put into the localization? Or, is it the character modifiers?

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -82,7 +82,8 @@ d_transylvania = {
 	allow = {
-		always = no
+		controls = 519
+		NOT = { has_landed_title = e_byzantium }
```
These old titular duchies should not be formable.

TurtleShroom replied:
@loup99 

Why? If they are there, why not use them? I tried to put them in places where they made sense.

loup99 replied:
It makes no sense to have Transylvania in the 5th century.

TurtleShroom replied:
You're right. I will fix that.

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -124,8 +125,9 @@ d_antioch = {
+		religion_group = christian
 		always = no
-	}	
+	}
```
These old titular duchies should not be formable.

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -205,6 +207,11 @@ d_jerusalem = {
+			religion = assyrian_pagan
+			religion_group = christian
+			controls = 774
+		}
```
These old titular duchies should not be formable.

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -264,6 +271,11 @@ d_sinai = {
+			religion_group = jewish_group
+			religion = bosporan_faith
+		}
+		controls = 788
```
These old titular duchies should not be formable.

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -444,6 +456,7 @@ d_normandy = {
 	culture = norman
 	allow = {
 		always = no
+		culture = norman
```
These old titular duchies should not be formable.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -691,11 +693,6 @@ title_despot = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```
Why did you remove these conditions?

TurtleShroom replied:
@loup99 

-because those conditions are stupid. If you control the Roman Empire, you should be able to use the Roman Minor Titles. _**Period.**_

loup99 replied:
These titles were related to religion.

TurtleShroom replied:
@loup99 

Those titles were Roman titles. The three true Roman Empires do not cease to be special because of the leader. In the hypothetical scenario that a follower of an unusual religion seizes the Roman Empire, he would most likely continue using Roman titles and traditions to legitimatize his rule. 
Doesn't everyone want to be Emperor of the Romans?

"Keeper of the King's Wardrobe" (etc.) is hardly a religious title. Caesar is a secular title used for the Roman Emperor. The only religious Roman title I can think is "Pontifex", which only Catholics and pagans used.

loup99 replied:
You forget that only certain religions can restore the roman titles. The whole mod's roman feature is based around some cultures and religious restrictions.

TurtleShroom replied:
You misunderstand. While it makes perfect sense that a Hindu or non-Latin/non-Hellenic character should not restore the Roman Empire, if a character of a foreign religion or culture USURPS the Roman throne, he should be able to use Roman features.

In other words, if a Hindu is somehow installed as Roman Emperor, he should be able to use the privileges and powers of a Roman Emperor. To do otherwise would be nonsensical, but allowing that same Hindu to _recreate_ the Roman Empire would also be nonsensical. I struck a fair balance.

loup99 replied:
They would use Hindu customs.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -735,11 +732,6 @@ title_sebastokrator = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```
Why did you remove these conditions?

TurtleShroom replied:
See above

loup99 replied:
See above.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -770,11 +762,13 @@ title_caesar = {
 			religion_group = christian
 			religion_group = jewish_group
 			religion_group = pagan_group
+			religion = bosporan_faith
```
Why should the Bosporans be allowed to give this title?

TurtleShroom replied:
@loup99 

Bosporans should give the title because Jews and Pagans should give the title.

loup99 replied:
Fair enough.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -816,11 +810,6 @@ title_kouropalates = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```
Why did you remove these conditions?

TurtleShroom replied:
See above

loup99 replied:
I don't see how the above is related.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -858,11 +847,6 @@ title_anthypatos = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```
Why did you remove these conditions?

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -898,13 +882,9 @@ title_imperial_eunuch = {
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
-		OR = {
```
Why did you remove these conditions?

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -1046,10 +1027,6 @@ title_court_musician = {
-		OR = {
-			religion_group = muslim
-			religion_group = indian_group
-		}
```
Why did you remove these conditions?

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -1410,17 +1391,17 @@ title_lawspeaker = {
-			is_feudal = yes
-			is_tribal = yes
-		}
-		OR = {
```
Why did you remove this?

TurtleShroom replied:
@loup99 

-because that specific title doesn't cease to exist if you're a republic. It's a religious title and thus transcends government among members of that faith.

loup99 replied:
You removed the "OR" condition too, and if you remove feudal and tribal, it won't make sense to have that title.

TurtleShroom replied:
@loup99 

I removed the "OR" under Feudal and Tribal because that would then allow any government to use that title. By removing that entire line, I removed government restrictions.

loup99 replied:
The OR was about religion. And I have explained for the governments.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2421,28 +2368,30 @@ title_yabgu = {
+					religion = tengri_pagan_reformed
+				}
+			}
+		}
```
These conditions should remain disabled.

TurtleShroom replied:
@loup99 

I disagree. The conditions I wrote are simple:

1. Any steppe nomad can use steppe nomadic titles.
2. Sedentary Tengri Pagans can use steppe nomadic titles as a part of their religious heritage.

This ensures that the titles can continue to be used when a Tengri Steppe Nomad settles down. It allows for greater flavor as a Tengri Pagan.

loup99 replied:
It also ensures that the title is distributed by the AI at unfitting occasions.

TurtleShroom replied:
@loup99 

What do you mean by "unfitting occasions"? How can handing out a Minor Title be "unfitting"?

loup99 replied:
Ahistorical or unplausible scenarios.

TurtleShroom replied:
Are you really going to say that handing out a Minor Title is ahistorical, but the entire Byzantine Empire going Hellenic, an Egyptian pantheon worshipping Hunnic Empire constructing pyramids. or Assyrian Paganism being ressurected, is acceptable?

loup99 replied:
The player can do whatever it wants, but there are limits. For instance what would you think if the ERE was allowed to distribute the honorary title of President of the US? Same thing here.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2478,29 +2427,31 @@ title_ishad = {
+					religion = tengri_pagan_reformed
+				}
+			}
+		}
```
These conditions should remain disabled.

TurtleShroom replied:
See above

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2535,29 +2486,31 @@ title_kundur = {
+					religion = tengri_pagan_reformed
+				}
+			}
+		}
```
These conditions should remain disabled.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2596,10 +2549,6 @@ title_master_of_the_royal_elephants = {
-		OR = {
-			is_feudal = yes
-			is_tribal = yes
-		}
```
These conditions should remain.

TurtleShroom replied:
@loup99 

Are you saying that an Indian cultured character ceases to use elephants as a republic? That's crazy. The conditions should not exist.

loup99 replied:
The title is "royal"

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2646,7 +2597,10 @@ title_royal_builder = {
+		OR = {
+			is_feudal = yes
+			is_merchant_republic = yes
+		}
```
It makes little sense that a Merchant republic would distribute this title.

TurtleShroom replied:
@loup99 

I disagree. I interpreted it as a royal architect of sorts: Merchant Republics build Trade Posts and Mansions, as well as upgrades to cities. The immense wealth a Republic gets should be administered by a minor title for flavor's sake. That said, it may have made more sense to use the Royal Architect title for this. If you'd like, I'll change it to that.

loup99 replied:
Add separate titles instead then.

TurtleShroom replied:
@loup99 

SHOOT, I didn't make the connection that Merchant Republics aren't royalty. I confused "nobility" with "royalty" again. I'll fix that immediately.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2743,16 +2697,16 @@ title_guru = {
 }
 title_genghis = {
-	dignity = 10
+	dignity = 500
```
That is far too much dignity.

TurtleShroom replied:
@loup99 

Genghis Khan was one of the greatest conquerers in human history. He created one of the largest empires the world had ever known. His title is the ultimate aspiration a steppe nomad can reach. He deserves that kind of Dignity.

loup99 replied:
This is not about Genghis Khan personally.

loup99 commented on WTWSMS/common/minor_titles/00_minor_titles.txt:
```diff
@@ -2743,16 +2697,16 @@ title_guru = {
-	monthly_salary = 0
-	monthly_prestige = 1.50
+	monthly_salary = 5
+	monthly_prestige = 5
```
That salary is far too significant.

TurtleShroom replied:
@loup99 

It represents tribute given to Genghis Khan and it represents the sheer amount of respect Genghis Khan should get if he achieves such a lofty feat.

Also, the Prestige should actually be higher than I set it. It should be at least as high as the Ceasar honorary title.

loup99 replied:
No, it isn't about about Genghis Khan personally, as above.

TurtleShroom replied:
@loup99 

It's about a person who _becomes this timeline's Genghis Khan_. That's a big deal. I'll half the Dignity, if that suits you. The lowest I'll go is one hundred.

loup99 replied:
No it isn't at all. It is solely a minor title.

TurtleShroom replied:
Do you mean that a player grants the title of Genghis to another?
I thought Mongols had a decision to _become_ Genghis Khan, and that gave them the Genghis title. That was what I based my coding on, here.

loup99 replied:
This is indeed the player granting a minor title to someone else, named "Gengis"

loup99 commented:
Good, otherwise I can take them out when I do the merge. The plan on Wahhabism is on hold, so there is nothing to discuss, which is why  I would like to keep it out of the main branch for now.

TurtleShroom commented:
@loup99 

Got it. I'll remove the Wahhabism additions. I'll add them back once you get the special plans underway. They can wait as long as you need them to, but I insist on adding them when its time is right.

loup99 commented:
We will discuss that in due time.

loup99 commented on ChangeLog.txt:
```diff
@@ -46,7 +46,10 @@ MINOR:
 - Redface characters can now expel Root West Holy Order
 - Root West Holy Order now has some epic camels
-- Wahhabi Islam flavor expansion
+- Insular characters are now more likely to send missionaries to foreigners
```
Where did you do this?

TurtleShroom replied:
@loup99 

It's in the Religion section. I used an increase to "AI_ZEAL" to force them to send Missionaries more often, and added the Always Convert Flag.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -292,7 +292,12 @@ k_angloland = {
+			culture = gepid
+		}
+		prestige = 2500
+		realm_size = 15
```
Should remain as solely always = no.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -1030,7 +1035,11 @@ k_wales = {
+			religion = catholic
+			religion = insular
+			religion = palagian
+		}
```
Should become always = no.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -1073,7 +1082,7 @@ e_carpathia = {
 	
 	tengri_pagan_reformed = 100
-	slavic_pagan_reformed = 50 # Crusade target weight
+	slavic_pagan_reformed = 50
```
That line is preferable to keep.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -563,7 +563,6 @@ d_scirii = {
 	color = { 41 155 129 }
 	
 	allow = {
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -584,7 +583,6 @@ d_iazyges = {
 	# tribe = yes
 	
 	allow = {
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -959,7 +957,6 @@ d_goths = {
 	color = { 123 123 10 }
 	
 	allow = {
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -1019,7 +1016,6 @@ d_ostrogoths = {
 		culture = ostrogothic
 		prestige = 500
 		controls = 456
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -1034,14 +1030,16 @@ d_thracian_goths = {
+			culture = crimean_gothic
 		}
 		controls = 497
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_tribes.txt:
```diff
@@ -1074,7 +1071,6 @@ d_rugii  = {
 	# tribe = yes
 	
 	allow = {
-		always = no
```
Keep that line but remove the rest.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -391,8 +391,6 @@ d_sky_lords = {
 	# Parent Religion 
 	religion = tengri_pagan_reformed
 	
-	culture = cuman
```
That line should be kept.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -966,10 +964,6 @@ d_great_company = {
-	# Parent Religion 
-	religion = germanic_pagan
-	culture = thuringian
-	
```
That line should be kept.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -451,7 +449,7 @@ d_warriors_perun = {
 	religion = slavic_pagan_reformed
 	
-	culture = russian
+	culture = old_slavic
```
Makes sense.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -1175,8 +1169,6 @@ d_navarrese_company = {
 	mercenary = yes
 	
-	# Parent Religion 
-	religion = hellenic_pagan
```
That line should be kept.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -2205,6 +2196,7 @@ d_mamluks = {
 	# Hire Trigger
 	allow = {
 		religion_group = muslim
+		year = 800
```
Why did you add this trigger? That will have to be reviewed.

TurtleShroom replied:
I added the year trigger because the Mamluks did not exist until after the Sunni/Shia split. The Mamluk slave brigades, as we know them, did not exist right at the dawn of Islam.

loup99 replied:
I know that, but that year isn't a good solution in my opinion.

loup99 commented on WTWSMS/common/landed_titles/old_duchies.txt:
```diff
@@ -83,6 +83,7 @@ d_transylvania = {
 	capital = 519 # Feher
 	allow = {
 		controls = 519
+		year = 850
```
This title should be always `= no

TurtleShroom commented:
@loup99 

I'm ready for implementation when you are. You can skip the titles if you must, but the idea of spawning titular titles creating bugs is silly. It doesn't bug out when the Hunnic Empire, Avaria, or Turkey is spawned...

loup99 commented:
It doesn't create bugs, it creates situationms which we don't want (people recreating titles of tribes randomly when they have already settled). The Migrational Era did not restart every 50 year throughout the Millenium playable.

loup99 commented:
Done.
